easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-2.8.0-gompi-2019b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenCoarrays'
+version = '2.8.0'
+
+homepage = 'https://github.com/sourceryinstitute/opencoarrays'
+description = """OpenCoarrays is an open-source software project that supports
+the coarray Fortran (CAF) parallel programming features of the Fortran 2008
+standard and several features proposed for Fortran 2015 in the draft Technical
+Specification TS 18508 Additional Parallel Features in Fortran."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/sourceryinstitute/opencoarrays/releases/download/%(version)s']
+sources = [SOURCE_TAR_GZ]
+patches = ['OpenCoarrays-2.2.0-cafrun-OpenMPI-mpiexec.patch']
```

Comment:
@furstj Please use the macro here, as in previous versions.

Reply:
@verdurin An existing patch that was created for version 2.2.0 is being used here, so `%(version)s` can't be used here...

Reply:
Ah, okay. That explains it. Thanks @boegel